pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGeneratorTest.java

```diff
@@ -150,12 +152,12 @@ public void simpleInput() throws Exception {
 
     // TODO: Remove once we move to a baseline of 2.90 or later
```

Comment:
@res0nance 
We've moved to a baseline way past 2.90.  Should this be removed?

Reply:
Let me reword it